.gitignore

```diff
@@ -33,4 +33,3 @@ nuget.exe
 *launchSettings.json
 **/Resources/*.Designer.cs
 .vscode/
-global.json
```

Comment:
After looking closer at https://github.com/aspnet/BuildTools/pull/661, I think you should split the netcoreapp2.2 update and global.json into separate pieces of work. Let's do the netcoreapp2.2 work first (I think @pranavkm is working on that already, maybe) and address the design issues on https://github.com/aspnet/BuildTools/pull/661 before we start committing the global.json file to source